package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-deploy-extensions",
-  "version": "1.2.3",
```

Comment:
Can we keep this versioning in sync with the extension specific ones? I was a bit thrown off when I saw a patch version bump but had dependency changes.

Reply:
Extensions versions are:
```
### auth0-bitbucket-deploy v2.8.0
### auth0-github-deploy v2.8.0
### auth0-gitlab-deploy v2.9.0
### auth0-visualstudio-deploy v2.7.0
```
I'll bump minor, but not sure how we could sync package version with all those extensions versions.